commands/sandbox.go

```diff
@@ -132,12 +139,26 @@ func RunSandboxUninstall(c *CmdConfig) error {
 
 // The connect command
 func RunSandboxConnect(c *CmdConfig) error {
-	err := ensureOneArg(c)
+	var arg string
+	var token bool
+	var creds do.SandboxCredentials
+	var err error
```

Comment:
```suggestion
	var (
		arg   string
		token bool
		creds do.SandboxCredentials
		err   error
	)
```

Just an FYI, you'll more frequently see multiple variables being declared like this in our Go code bases.